OurUmbraco/Our/Examine/ExamineHelper.cs

```diff
@@ -60,12 +60,16 @@ public static SimpleDataSet MapFileToSimpleDataIndexItem(FileInfo file, SimpleDa
             simpleDataSet.RowData.Add("updateDate", file.CreationTime.ToString("yyyy-MM-dd HH:mm:ss"));
             simpleDataSet.RowData.Add("nodeTypeAlias", "documentation");
 
-            simpleDataSet.RowData.Add("dateCreated", file.CreationTime.ToString("yyyy-MM-dd HH:mm:ss"));
-
-            //TODO: This will always be exactly the same since all files are written at the same time IIRC
+            simpleDataSet.RowData.Add("dateCreated", file.CreationTime.ToString("yyyy-MM-dd HH:mm:ss"));           always be exactly the same since results all files are written at the same time IIRC
```

Comment:
Thanks @OwainWilliams ! I think you messed up a code comment here though ;-)

Reply:
Oops! Not sure how that happened. My local copy has the comment correct. 
I'll fix it and resubmit.